ts/Series/HeikinAshi/HeikinAshiPoint.ts

```diff
@@ -0,0 +1,148 @@
+/* *
+ *
+ *  (c) 2010-2021 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+import HeikinAshiSeries from './HeikinAshiSeries';
+
+const {
+    seriesTypes: {
+        candlestick: CandlestickSeries
+    }
+} = SeriesRegistry;
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+class HeikinAshiPoint extends CandlestickSeries.prototype.pointClass {
+
+    /* *
+     *
+     *  Properties
+     *
+     * */
+
+    public series: HeikinAshiSeries = void 0 as any;
+
+    /* *
+     *
+     *  Functions
+     *
+     * */
+
+    /**
+     * Translate the values to pixels.
+     * @private
+     *
+     * @function Highcharts.seriesTypes.heikinashi#doTranslation
+     *
+     * @return {void}
+     *
+     */
+    public doTranslation(): void {
+        const point = this,
+            series = point.series,
+            yAxis = series.yAxis,
+            translated = [
+                'plotOpen',
+                'plotHigh',
+                'plotLow',
+                'plotClose',
+                'yBottom'
+            ]; // translate OHLC for
+
+        [point.open, point.high, point.low, point.close, point.low]
+            .forEach(
+                function (value, i): void {
+                    if (value !== null) {
+                        (point as any)[translated[i]] =
```

Comment:
Does it have to be `as any`? Can't we add a TS definition for `point.translated`?

Reply:
It's not that easy to get rid of that `any`. 
Additional after some research, I see that we use that approach in other similar cases like this one so I guess for now it has to stay like this.